ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-segment-security.j2

```
@@ -0,0 +1,80 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - router segment-security #}
+{% if router_segment_security is arista.avd.defined %}
+!
+router segment-security
+{%     if router_segment_security.enabled is arista.avd.defined(true) %}
+   no shutdown
+{%     endif %}
+{%     if router_segment_security.policies is arista.avd.defined %}
+{%         for policy in router_segment_security.policies %}
+{%             if policy.name is arista.avd.defined %}
+   !
+   policy {{ policy.name }}
+{%                 if policy.sequence_numbers is arista.avd.defined %}
+{%                     for sequence in policy.sequence_numbers | arista.avd.natural_sort('sequence') %}
+{%                         if sequence.sequence is arista.avd.defined and sequence.action is arista.avd.defined and sequence.application is arista.avd.defined %}
+{%                             set eos_cli = sequence.action %}
+{%                             if sequence.action == "redirect" %}
+{%                                 if sequence.next_hop is arista.avd.defined %}
+{%                                     set eos_cli = eos_cli ~ " next-hop " ~ sequence.next_hop %}
+{%                                 else %}
+{%                                     set eos_cli = "BAD!" %}
```

? If next-hop is required for action `redirect`, we should add that to the `if` checking for `redirect`. If it fails, we silently ignore the redirect. That is what we do elsewhere in `eos_cli_config_gen`.
Would be good to make it clear in the schema description for `action` that for `redirect` `next-hop` must be set.
You can also just decide to skip the sequence like you do now, by simply adding a `continue` in this case.